gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -87,7 +87,7 @@ def encode(self, variant):
       columns.append(encoded_calls)
     columns = [self._encode_value(c) for c in columns]
 
-    return '\t'.join(columns) + '\n'
+    return ('\t'.join(columns) + '\n').encode('utf-8')
```

Comment:
Please update the type hint of this method.

Also in all the previous files that we replaced str with byte please check the type hints and update them.

Reply:
SG, but please double check if I miss any.